src/hotspot/share/memory/arena.cpp

```diff
@@ -203,34 +168,35 @@ void* Chunk::operator new (size_t sizeofChunk, AllocFailType alloc_failmode, siz
   assert(sizeofChunk == sizeof(Chunk), "weird request size");
   assert(is_aligned(length, ARENA_AMALLOC_ALIGNMENT), "chunk payload length misaligned: "
          SIZE_FORMAT ".", length);
+  // Try to reuse a cached chunk from the pool
```

Comment:
Can you modify this comment as:
// Try to reuse a freed chunk from the pool
For some reason, I was confused about which chunks were on the chunk pool.